ui/src/image_annotator.tsx

```diff
@@ -468,6 +474,13 @@ export const XImageAnnotator = ({ model }: { model: ImageAnnotator }) => {
         { cursor_x, cursor_y } = eventToCursor(e, rect, zoom, imgPositionRef.current),
         intersected = getIntersectedShape(drawnShapes, cursor_x, cursor_y)
 
+      if (model.events?.includes('click') && start?.x && start?.y) {
```

Comment:
We probably want to support picking `[0,0]` as well.

```suggestion
      if (start && model.events?.includes('click')) {
```